Makefile

```diff
@@ -39,3 +39,7 @@ buster-slim: 12-buster-slim 14-buster-slim 16-buster-slim
 16-buster-slim:
 	docker build -t local/articulate-node:16-buster-slim 16-buster-slim
 .PHONY: 16-buster-slim
+
+16-bullseye-slim:
```

Comment:
we should add a new `bullseye-slim` target and then also make sure that gets added to `all`